data/1H-pyrrole.yml

```diff
@@ -0,0 +1,19 @@
+# 1H-pyrrole
+A: 9130.64795
+B: 9001.35664
+C: 4532.11989
+Delta_J: 1.129e-03
+Delta_JK: 3.836e-03
+Delta_K: 2.717900869e-24
+del_J: 0.010157300e-24
+del_K: 6.135949147e-24
+chi_aa: 1.40475
+chi_bb: 1.29577
+u_a: 1.74
+rep: "Ir"
+name: "1H-pyrrole"
+formula: "C4H5N"
+smiles: "C1=CNC=C1"
+notes: "dipole moment from doi.org/10.1016/0022-2860(69)80031-1, Delta_K much larger on cdms"
+doi: "doi.org/10.1021/acs.jpca.0c02919"
```

Comment:
I don't think this is a good fit in their discharge paper...
Here's a better source for the molecular constants: 10.1016/0022-2852(88)90019-7
And for the hfs constants: 10.1016/0022-2860(69)80031-1
In the first paper, they report a value for R6, but I believe this should be identical to del_K. Also they use IIIr, so just flip the signs of del_J and del_K and call the rep IIIl instead.